airflow/ui/src/components/Clear/ClearAccordion.tsx

```diff
@@ -35,7 +35,7 @@ type Props = {
 // Table is in memory, pagination and sorting are disabled.
 // TODO: Make a front-end only unconnected table component with client side ordering and pagination
 const ClearAccordion = ({ affectedTasks, note, setNote }: Props) => (
-  <Accordion.Root collapsible defaultValue={["note"]} multiple={false} variant="enclosed">
+  <Accordion.Root collapsible defaultValue={["tasks"]} multiple={false} variant="enclosed">
```

Comment:
That was the default behavior but Jens explicitly asked to show the `note` as default to encourage the usage of notes going forward.  I'm neutral on what should be the default. (tasks feels more natural, but encouraging usage of notes, is not bad too).

Reply:
I get that argument but I think seeing affected tasks is a must-have for clearing anything while adding a note is a nice to have.

Reply:
Ok, lets go with `tasks` as default. I'll do the same or `mark task as`.